shared/common-adapters/markdown.stories.js

```diff
@@ -1,88 +1,102 @@
 // @flow
+import * as I from 'immutable'
 import * as React from 'react'
 import * as Sb from '../stories/storybook'
-import Markdown from './markdown'
+import * as Kb from './index'
+import Markdown, {parserFromMeta, MarkdownMeta} from './markdown'
+import OriginalParser from '../markdown/parser'
+import RandomSeed from 'random-seed'
+import RandExp from 'randexp'
```

Comment:
randexp seems cool and useful, we already have a PRNG, so maybe we should use that instead of a new dep. could add a string helper function in it.
https://github.com/keybase/client/blob/master/shared/stories/storybook.shared.js#L134

Reply:
also might as well move randexp into there too